packages/victory-brush-container/src/brush-helpers.ts

```diff
@@ -1,8 +1,26 @@
 import { Selection } from "victory-core";
 import { assign, throttle, isFunction, defaults, mapValues } from "lodash";
 import isEqual from "react-fast-compare";
+export interface BrushHelpers {
+  getDimension(props): any;
+  withinBounds(point, bounds, padding?): any;
+  getDomainBox(props, fullDomain, selectedDomain?): any;
+  getHandles(props, domainBox): any;
+  getActiveHandles(point, props, domainBox): any;
+  getResizeMutation(box, handles): any;
+  getMinimumDomain(): any;
+  getDefaultBrushArea(targetProps, cachedDomain, evt): any;
+  getSelectionMutation(point, box, brushDimension): any;
+  panBox(props, point): any;
+  constrainBox(box, fullDomainBox): any;
+  constrainPoint(point, fullDomainBox): any;
+  hasMoved(props): any;
+  onMouseDown(evt, targetProps): any;
+  onGlobalMouseMove(evt, targetProps): any;
+  onGlobalMouseUp(evt, targetProps): any;
+}
```

Comment:
I don't think we need this interface ... the exported `const BrushHelpers` will automatically be able to infer this type from the implementation.